classes/template/PKPTemplateManager.inc.php

```diff
@@ -766,6 +789,21 @@ function clearTemplateCache() {
 	}
 
 	/**
+	 * Clear all compiled CSS files
+	 */
+	public function clearCssCache() {
+		$cacheDirectory = CacheManager::getFileCachePath();
+		$files = scandir($cacheDirectory);
```

Comment:
I think you could probably use

    array_map('unlink', glob(CacheManager::getFileCachePath . DIRECTORY_SEPARATOR . '*.' . CSS_FILENAME_SUFFIX);

A little more compact and probably a little faster, for what it's worth.